jira/jira.go

```diff
@@ -165,18 +188,18 @@ func getDescription(ctx *ankh.ExecutionContext, env string) (string, error) {
 	}
 
 	// Otherwise, prompt for message
-	message, err := promptForDescription(ctx.Chart, ctx.DeploymentTag, env)
+	message, err := promptForDescription(chartName, *chart.Tag, envOrContext)
 	if err != nil {
 		ctx.Logger.Infof("Unable to prompt for description. Will use default description")
 	}
 
 	return message, nil
 }
 
-func promptForSummary(chart string, version string, env string) (string, error) {
-	defaultSubject := fmt.Sprintf("Deployment of %v verson:%v to *%v*", chart, version, env)
-	if env == "rollback" {
-		defaultSubject = fmt.Sprintf("Rollback of %v in *%v*", chart, env)
+func promptForSummary(chart string, version string, envOrContext string) (string, error) {
+	defaultSubject := fmt.Sprintf("Deployment of %v verson:%v to *%v*", chart, version, envOrContext)
+	if envOrContext == "rollback" {
+		defaultSubject = fmt.Sprintf("Rollback of %v in *%v*", chart, envOrContext)
```

Comment:
i see this is how it was before, but this will make the string say `.. in *rollback*` as the default, which probably isn't what we want

Reply:
Yeah I'm confused how the environment/context would be "rollback"... Were you thinking of simply removing the `"in *%v*"` part?

Reply:
i think this may be an existing bug. Let's address it in a separate changeset.